### DISCUSSION TITLE
Option To Exclude Steam Overlay From Game Capture?

### DISCUSSION THREAD

**Discussion by BananaManner** [obsproject/obs-studio#9793] [Development Questions]
It may sound a little redundant as there's already the options to capture 3rd party overlay's such as Steam in the options but short of using Display Capture this is the only way to have post processing be captured by some effects in ReShade or even Nvidia's own FXAA, so having that be an option would be helpful especially for privacy related reasons.

**Comment by matoi974** [comment]
Disable the Steam overlay if you don't want to show it, obs-studio's overlay capture is inconsistent.

**Comment by BananaManner** [comment]
I shouldn't have to disable it if I use it though, and avoiding the problem by disabling the overlay isn't a solution.

**Comment by notr1ch** [comment]
There is no defined order for which multiple injected hooks are called in, the option to capture overlays attempts this already but it's pretty much random luck whether it works (depends on whether OBS hooks before Steam or ReShade etc).

**Comment by BananaManner** [comment]
If I may ask, is there a potential way to address the random luck aspect without a lot of work or would it require a serious amount of effort?
